read_zdf_without_zivid.py

```diff
@@ -24,14 +24,16 @@ def _main():
         contrast = data["data"]["contrast"][:, :]
 
     # Displaying the RGB image
+    plt.figure()
     plt.imshow(rgb)
-    plt.show()
+    plt.title("RGB image")
 
     # Displaying the Depth Image
     Z = xyz[:, :, 2]
+    plt.figure()
     plt.imshow(Z, vmin=np.nanmin(Z), vmax=np.nanmax(Z), cmap="jet")
     plt.colorbar()
-    plt.show()
```

Comment:
why is show removed?

Reply:
It is not required. I think plt.imshow() already does that.

Reply:
Actually, it was. This is why it didn't show up from the cmd. In Spyder it shows without it.

Reply:
So is this going to be reverted or not?

Reply:
we need to have plt.show() - the current version shows images.